walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,124 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
+    public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+        Collection<TimeRange> workingTimes = new ArrayList<TimeRange>();
+        int start = TimeRange.START_OF_DAY;
+        int end;
+        boolean foundGuest = false;
```

Comment:
You should move this declaration inside the for loop, that way you don't have to reset its value at the end of each loop. It is only used inside the loop itself so there it doesn't need to be defined outside of it.